chainer/functions/normalization/layer_normalization.py

```diff
@@ -34,45 +36,52 @@ def check_type_forward(self, in_types):
             gamma_type.shape == beta_type.shape,
         )
 
+    def _compute(self, xp, x):
```

Comment:
Could you add the comment that xp can be `chainer.functions` as you did above.

Reply:
Fixed.